simple/schema.py

```diff
@@ -111,203 +132,322 @@ class Gravity(enum.Enum):
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
+
+    parameter = Column(
+        String(30),
+        ForeignKey("Parameters.parameter", onupdate="cascade"),
+        primary_key=True,
+    )
     value = Column(Float, nullable=False)
     value_error = Column(Float)
     unit = Column(String(20))
     comments = Column(String(1000))
-    reference = Column(String(30), ForeignKey('Publications.reference', onupdate='cascade'), primary_key=True)
+    reference = Column(
+        String(30),
+        ForeignKey("Publications.reference", onupdate="cascade"),
+        primary_key=True,
+    )
 
 
 class CompanionRelationships(Base):
     # Table to store information about companions
     __tablename__ = 'CompanionRelationships'
-    source = Column(String(100), ForeignKey('Sources.source', ondelete='cascade', onupdate='cascade'),
-                    nullable=False, primary_key=True)
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
     companion_name = Column(String(100), nullable=False, primary_key=True)
     projected_separation_arcsec = Column(Float)
     projected_separation_error = Column(Float)
-    relationship = Column(String(100), nullable=False) # Relationship of source to companion. 
-                                                       # Options: Child, Sibling, Parent, Unresolved Parent
+    relationship = Column(String(100), nullable=False)
+    # Relationship of source to companion.
+    # Options: Child, Sibling, Parent, Unresolved Parent
     comments = Column(String(1000))
-    reference = Column(String(30), ForeignKey('Publications.reference', onupdate='cascade'))
-    other_companion_names = Column(String(10000)) # other names of the companions 
+    reference = Column(
+        String(30), ForeignKey("Publications.reference", onupdate="cascade")
+    )
+    other_companion_names = Column(String(10000))  # other names of the companions
 
 # -------------------------------------------------------------------------------------------------------------------
 # Views
 
+
 ParallaxView = view(
     "ParallaxView",
     Base.metadata,
     sa.select(
-        Parallaxes.source.label('source'),
-        Parallaxes.parallax.label('parallax'),
-        Parallaxes.parallax_error.label('parallax_error'),
-        (1000./Parallaxes.parallax).label('distance'),  # distance in parsecs
-        Parallaxes.comments.label('comments'),
-        Parallaxes.reference.label('reference')
-    ).select_from(Parallaxes)
-    .where(sa.and_(Parallaxes.adopted == True, Parallaxes.parallax > 0)),
+        Parallaxes.source.label("source"),
+        Parallaxes.parallax.label("parallax"),
+        Parallaxes.parallax_error.label("parallax_error"),
+        (1000.0 / Parallaxes.parallax).label("distance"),  # distance in parsecs
```

Comment:
what happens if parallax is 0, NULL or negative, here?

Reply:
Again, that's beyond the scope of this PR.  I've addded this to #474 .